### DISCUSSION TITLE
Gerätetypen Übersicht Standorte mit anbieten

### DISCUSSION THREAD

**Discussion by moritzvolkmar** [Samedis-care/samedis-care-community#5] [Idea retired / denied]
In der Gerätetypen Übersicht wäre es sinnvoll neben den Abteilungen auch die Standorte noch auswählen bzw. filtern zu können.

![Bildschirmfoto 2022-05-31 um 14 11 51](https://user-images.githubusercontent.com/80519295/171170732-3d488efa-8340-4f15-83ee-729f0ca087e3.png)

**Comment by YvesR** [comment]
Would not work, the data shows device models, not inventory

**Reply by moritzvolkmar** [reply]
Why? Departments can also be selected/filtered!
